e2e/react/src/react-module-federation.test.ts

```diff
@@ -40,7 +40,7 @@ describe('React Module Federation', () => {
         const remote3 = uniq('remote3');
 
         runCLI(
-          `generate @nx/react:host ${shell} --remotes=${remote1},${remote2},${remote3} --e2eTestRunner=cypress --style=css --no-interactive --skipFormat --js=${js}`
+          `generate @nx/react:host ${shell} --remotes=${remote1},${remote2},${remote3} --bundler=webpack --e2eTestRunner=cypress --style=css --no-interactive --skipFormat --js=${js}`
```

Comment:
should test in the scenario in rspack too?

Reply:
there's a separate test file testing rspack